docs/applications/mobu/configuring.rst

```diff
@@ -77,15 +77,22 @@ Here is a more complex example that runs a set of notebooks as a test:
          options:
            repo_url: "https://github.com/lsst-sqre/system-test.git"
            repo_branch: "prod"
+           exclude_dirs: ["some/experiment/dir", "another/dir"]
            max_executions: 1
          restart: true
 
 Here, note that the UID and primary GID for the user are specified, so this example will work in deployments that do not use Firestore and synthesized user private groups.
 
-This uses the business ``NotebookRunner`` instead, which checks out a Git repository and runs all notebooks at the top level of that repository.
+This uses the business ``NotebookRunner`` instead, which checks out a Git repository and runs all notebooks in that repository, at the top level and any directories and subdirectories, except:
+
+* Notebooks in the ``mobu_exclude`` directory will never be executed.
+* ``options.exclude_dirs`` tells mobu to not excecute any notebooks in those directories or any descendant directories.
```

Comment:
You have this line twice, here and below. I think delete this one and keep the one below?